src/Illuminate/Notifications/Console/stubs/notifications.stub

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateNotificationsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notifications', function (Blueprint $table) {
+            $table->string('id')->primary();
+            $table->string('type');
+            $table->string('notifiable_type');
+            $table->integer('notifiable_id');
```

Comment:
I had originally, but I presumed that was what @taylorotwell was referring to with it not matching doc blocks as the documentation for notifications doesn't use the morphs shorthand.